runtime.requestUpdateCheck() argument in example correction (#29715)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/runtime/requestupdatecheck/index.md

### Return value

> A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that is fulfilled with two arguments:

Reviewer: ```suggestion
A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that is fulfilled with an object with the result of the update request.
```

The previous and current documentation is incorrect. It's a single object, not a number of positional parameters.

> - `status`
>   - : A {{WebExtAPIRef('runtime.RequestUpdateCheckStatus')}} value — the result of the update check.
> - `details` {{optional_inline}}

Reviewer: ```suggestion
- `result` {{optional_inline}}
  - `status`
    - : A {{WebExtAPIRef('runtime.RequestUpdateCheckStatus')}} value — the result of the update check.
```

Changes:
- moved "status" under the object.
- renamed "details" to "result", consistent with Chrome's docs.

  - : `object`. If `status` is `update_available`, this contains more information about the update. It is an object containing a property:

    - `version`
      - : `string`. The update's version.

## Examples

Request an update and log the new version if one is available:

```js

> function onRequested(status, details) {

Reviewer: ```suggestion
function onRequested(result) {
```

Github doesn't offer more context, more changes that you need to make before merging:

- Replace `status` with `result.status` (2x)
- Replace `details.version` with `result.version`
